CONTRIBUTING.rst

```diff
@@ -1,56 +1,55 @@
-# James Webb Space Telescope Data Analysis Tool Notebooks
+============================
+Contribute to JDAT Notebooks
+============================
 
-   ``jdat_notebooks`` are one component of STScI's larger [Data Analysis Tools Ecosystem](https://jwst-docs.stsci.edu/jwst-post-pipeline-data-analysis).
-
-   The [jdat_notebooks repository](https://github.com/spacetelescope/jdat_notebooks) is for already-reviewed notebooks that are ready for consumption by the science community.  If you wish to contribute notebooks or technical expertise, we refer you to [dat_pyinthesky](https://github.com/spacetelescope/dat_pyinthesky/tree/master/jdat_notebooks) as the place to submit a first draft.  The description below outlines the whole process, which begins in `dat_pyinthesky` but finishes in `jdat_notebooks`.
-
-
-# Development Procedure for JDAT Notebooks
-
-This document is a description of the JWST Data Analysis Tools (JDAT) Approach to
-"Notebook-Driven Development".  The procedures here outline the process for
+This document outlines the process for contributing code to the jdat notebooks by following the JWST Data Analysis Tools (JDAT) Approach to
```

Comment:
This document outlines the process for contributing code to the Data Analysis Tools Ecosystem by following the JWST Data Analysis Tools (JDAT) Approach to